guides/concepts/list-of-operations.md

```diff
@@ -253,11 +256,16 @@ Possible errors:
 ## Set Options
 [JavaScript](http://stellar.github.io/js-stellar-sdk/Operation.html#.setOptions) | [Java](http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/SetOptionsOperation.Builder.html) | [Go](https://godoc.org/github.com/stellar/go/txnbuild#SetOptions)
 
-This operation sets the options for an account.
+Sets options for an account, such as setting the inflation destination or adding an additional
+signer on an account.
+
+This operations allows you to set multiple options on an account in a single operation, such as
+changing a operation threshold and setting the flags on an account at the same time.
 
-For more information on the signing options, please refer to the [multi-sig doc](./multi-sig.md).
+For more information on the options related to signing, please refer our documentation on
```

Comment:
Could consider being terser: `For more information on the options related to signing, see [multi-sig](./multi-sig.md).`